### ISSUE TITLE
Invalid Forecast/Advisory link

### ISSUE THREAD

**Issue by timtrice** [ropensci/rrricanes#54]
For EP031999, 

```
10.
stop(sprintf("Invalid Forecast/Advisory link. %s", link)) at fstadv.R#202
9.
.f(.x[[i]], ...) 
8.
purrr::map(filter_fstadv(products), fstadv) at fstadv.R#157
7.
get_fstadv(url) at check-errors.R#33
6.
.f(.x[[i]], ...) 
5.
map(.x = nums, .f = function(x) {
    url <- storms %>% slice(x) %>% .$Link
    key <- storms %>% slice(x) %>% .$Key
    obs.fstadv <- get_fstadv(url) ... at check-errors.R#30
4.
eval(ei, envir) 
3.
eval(ei, envir) 
2.
withVisible(eval(ei, envir)) 
1.
source("~/Projects/Hurricanes/data-raw/check-errors.R") 
```

**Comment by timtrice** [comment]
In `fstadv`, function searches contents of product for "MIATCMAT" and "MIATCMEP" in header. These are the forecast/advisory headers for cyclones in the Atlantic ("AT") and east Pacific ("EP"). 

However, for cyclone EP041999 the system enters the central Pacific basin. The header of the forecast/advisory product changes to

```
WTPA24 PHNL 242100
TROPICAL DEPRESSION FOUR-C FORECAST/ADVISORY NUMBER 7
NATIONAL WEATHER SERVICE HONOLULU HI   EP0499
2100Z SAT JUL 24 1999
```

Note the "WTPA24".  The "4" is the wallet package (there are 5 and get recycled throughout the seaon, in order). 

Seems, need to add "WTPA2" to the check to resolve this issue.

**Comment by timtrice** [comment]
Modified `fstadv` to accept "WTPT2" in header.

Modified `scrape_key` to accept optional "NATIONAL WEATHER SERVICE HONOLULU HI".

All relevant tests pass.

Resolved in commit 83e394e

**Comment by timtrice** [comment]
Modified pattern per commit 83e394e. 

[Hurricane Dora, Advisory 57](http://www.nhc.noaa.gov/archive/1999/mar/MEP0799.057.html) only uses "WTPA" in the header.

[Hurricane Eugene Advisory 23](http://www.nhc.noaa.gov/archive/1999/mar/MEP0899.023.html) uses "TPA" in the header.

Modified pattern to accomodate optional "W"; removed the numeric 2 at the end.
